docs/index.mdx

```diff
@@ -1,6 +1,7 @@
 export { future as theme } from '../src/themes'
-import { Image, Notes, Appear } from '../src'
+import { Image, Notes, Appear, Code } from '../src'
 import Layout from './Layout'
+export highlight from 'react-syntax-highlighter/styles/prism/solarizedlight'
```

Comment:
This should be in the theme, shouldn't it?

Reply:
This is an overriding example.